Parsers/Creator Toolbox.js

```diff
@@ -0,0 +1,10 @@
+/*
+activation_example:!ctb
+regex:(!ctb)||(!creatortoolbox)||(!toolbox) 
```

Comment:
Please remove double pipes in regex or else this will run for all messages. regen only needs single |

Reply:
I've removed the double pipes